library/cwm/src/lib/cwm/wrapper_widget.rb

```diff
@@ -0,0 +1,46 @@
+require "yast"
+
+require "cwm/abstract_widget"
+
+module CWM
+  # Wrapper when combination of old hash based CWM definition needed to be used
+  # with new object based one. Useful e.g. when content is provided by other
+  # module like CWMFirewallInterfaces.CreateOpenFirewallWidget
+  # @note it does not support some of common methods and helpers from abstract widget
+  #
+  # @example how to initialize object from firewall interface
+  #   ::CWM::WrapperWidget.new("firewall",
+  #     CWMFirewallInterfaces.CreateOpenFirewallWidget("services" => ["service:sshd", "service:ntp"])
+  #   )
+  class WrapperWidget < AbstractWidget
+    # Creates new instance with specified id and content
+    # @param widget_id [String] name of widget used as identified, have to be unique.
+    #   It have to be same as real widget id in content, otherwise enable/disable won't work.
+    # @param content [Hash<String, Object>] CWM hash definition
```

Comment:
In fact I prefer to have here `StringHash`. I am not sure if it will work with widgets inside and what is more important I really do no want to support.